src/MotoROS.h

```diff
@@ -101,6 +101,7 @@
 #include "Tests_CtrlGroup.h"
 #include "Tests_TestUtils.h"
 #include "Tests_RosMotoPlusConversionUtils.h"
+#include "Tests_ControllerStatusIO.h"
```

Comment:
The source and header for this has not been added to the VS project.

Reply:
That was on purpose, as these tests still hang my YRC.

Reply:
I guess you posted this as it breaks the MSBuild build.

That's indeed problematic. Will fix/work-around.